### DISCUSSION TITLE
Release process

### DISCUSSION THREAD

**Discussion by nurey** [themesberg/flowbite-react#1429] [General]
I am wondering what the release process is for this project. Forgive me if this is documented somewhere. I could not find it in the README. I see the last release was April 16. There's some fixes that have gone in that I would love to benefit from. I would love to know when the next release will be cut. Thank you!

**Comment by SutuSebastian** [comment]
It is not documented anywhere, but its a good idea to do so.
Initially I was planning to do a release at the end of each week (if we have any batch of updates), but slowly I got overwhelmed by work. Now that I have more free time on my hands I'll be pushing some more on the library, issues, PR's, discussions, etc.

Regarding how a release is done:
We are using [Changesets](https://github.com/changesets/changesets) so u can read more on their repo how things work, pretty much is no-code release, just a push of a button on the [Version Packages](https://github.com/themesberg/flowbite-react/pull/1373) PR, there u can find the summary of all the changes that will be brought to each package/namespace.
